src/pages/components/button/usage.mdx

```diff
@@ -102,186 +132,419 @@ import { Add16 } from "@carbon/icons-react";
+
+</InlineNotification>
 
 <Row>
 <Column colLg={8}>
 
-![button with glyph](images/button-usage-1.png)
+![Button groups in context](images/button_usage_23.png)
 
-<Caption>Glyph usage in buttons</Caption>
+</Column>
+</Row>
+
+#### Stacked button groups
+
+Typical landing pages for product have buttons side by side. However vertical button groups are also common in products, to save real estate in narrow columns and occasionally side panels. In these instances, the primary button is always on top and the secondary or tertiary button is below.
+
+Generally, when designers stack buttons, they tend to use the fixed buttons. However, stacked fluid buttons are also an option in desktop side panels with especially long calls to action. 
+
+<Row>
+<Column colLg={12}>
+
+![Fluid and fixed stacked button group examples](images/button_usage_24.png)
 
 </Column>
 </Row>
 
-## Danger button usage
+#### Groups with Danger buttons
+
+Danger buttons have a different visual style to inform users of potentially destructive actions they are about to take. Within a set, the danger button should be styled as a primary button.
+
+<Row>
+<Column colLg={8}>
+
+![Danger button treated as Primary button in a modal](images/button_usage_25.png)
+
+</Column>
+</Row>
+
+## Content
+
+### Text labels
+
+A button’s text label is the most important element on a button, as it communicates the action that will be performed when the user interacts with it. Buttons need to be clear and predictable.
+
+Button labels should clearly indicate the action of the button. To provide enough context, use the {verb} + {noun} content formula on buttons except in the case of common actions like “Done”, “Close”, “Cancel”, “Add”, or “Delete.”
+
+There are exceptions to this rule for situations in which button length could cause problems in compact UIs or negatively impact translation, but the {verb} + {noun} formula is still best practice.
+
+For consistency, see Carbon’s [content guidelines](guidelines/content/overview) for a list of recommended action labels.
+
+<DoDontRow>
+  <DoDont caption="Do use the {verb} + {noun} content formula in buttons whenever possible.">
+
+![Do use the verb + noun content formula in buttons whenever possible](images/button_usage_26.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not use only a noun as a button label.">
+
+![Do not use only a noun as a button label](images/button_usage_27.png)
+
+  </DoDont>
+</DoDontRow>
+
+## Behaviors
+
+### Interactions
+
+#### Mouse
+
+Users can trigger a button by clicking anywhere within the button container.
+
+#### Keyboard
+
+Users can trigger a button by pressing `Enter` or `Space` while the button has focus. For additional keyboard interactions, see the accessibility tab.
+
+#### Screen readers
+
+VoiceOver: Users can trigger a button by pressing `Enter` or `Space` while the button has focus.
```

Comment:
Wonder if we could simplify this to:

> For all screen readers, users can trigger a button by pressing `Enter` or `Space` while the button has focus.

Reply:
If you make a suggestion here I'd be happy to commit it, not sure how to answer that or change this.

Reply:
Would the suggestion I made above work? Or are you looking for something else?